### ISSUE TITLE
Unexpected Focus Movement After Dismissing an Issue

### ISSUE THREAD

**Issue by MajoMaldonado** [equalizedigital/accessibility-checker#1771]
Location / URL:https://edac-vpat.instawp.co/wp-admin/post.php?post=1639&action=edit&classic-editor__forget&classic-editor 
Screenshot: 

<img width="771" height="307" alt="Image" src="https://github.com/user-attachments/assets/d11b87bb-fd89-4924-95d2-583d6bd48700" />

WCAG S.C: 2.4.3 Focus Order (Level A)

Issue Description: When the Dismiss Issue button is activated, the corresponding issue is dismissed and the accordion collapses. After the content is updated, keyboard focus is automatically moved to the beginning of the results list instead of remaining at a logical location related to the user's previous interaction. This unexpected focus movement disrupts the navigation flow and requires keyboard and assistive technology users to navigate back to continue reviewing the remaining issues.
